www/src/examples/Overlays/PopoverContained.js

```diff
@@ -3,9 +3,9 @@ function Example() {
   const [target, setTarget] = useState(null);
   const ref = useRef(null);
 
-  const handleClick = ({ targ }) => {
+  const handleClick = event => {
```

Comment:
```suggestion
  const handleClick = ({ target }) => {
```

Reply:
What you suggest is what I did in first place, but then CI will fail because **"target"** is already defined in component scope (no-shadowed-variable rule).

Two possibilities : 
- do no destructure event into **"target"** variable
or
- rename **"target"** local state variable into something else